3.10/arangographml-getting-started.md

```diff
@@ -0,0 +1,395 @@
+                      '_key': '123',
+                      '_rev': '_gkUc8By--_',
+                      'run_id': '123',
+                      'test': {'accuracy': 0.8891242216547955,
+                               'confusion_matrix': [[13271, 2092],
+                                                    [1276, 5684]],
+                               'f1': 0.9,
+                               'loss': 0.1,
+                               'precision': 0.9,
+                               'recall': 0.8,
+                               'roc_auc': 0.8},
+                      'timestamp': '2023-09-01T17:32:05.899493',
+                      'validation': {'accuracy': 0.9,
+                               'confusion_matrix': [[13271, 2092],
+                                                    [1276, 5684]],
+                               'f1': 0.85,
+                               'loss': 0.1,
+                               'precision': 0.86,
+                               'recall': 0.85,
+                               'roc_auc': 0.85}},
+ 'target_collection': 'target_collection_name',
+ 'target_field': 'label_field'}
+```
+
+## Prediction
+
+After selecting a model, it is time to persist the results to a collection
+using the `predict` function.
+
+```
+prediction_spec = {
+              "project_name": project.name,
+              "database_name": db.name,
+              "model_id": model.model_id
+            }
+
+prediction_job = arangoml.prediction.predict(prediction_spec)
+print(prediction_job)
+```
+
+This creates a prediction job that grabs data and generates inferences using the
+selected model. Then, by default, it writes the predictions to a new collection
+connected via an edge to the original source document.
+You may choose to specify instead a `collection`.
+
+### Get prediction job status
+
+```
+prediction_status = arangoml.prediction.get_job(prediction_job.job_id)
+```
+
+```
+{'database_name': 'db_name',
+ 'job_id': '123-ee43-4106-99e7-123',
+ 'job_state_information': {'outputAttribute': 'label_field_predicted',
+                           'outputCollectionName': 'collectionName_predicted_123',
+                           'outputGraphName': 'graph_name'},
+ 'job_status': 'COMPLETED',
+ 'model_id': '123',
+ 'project_id': '123456',
+ 'project_name': 'project_name',
+ 'time_ended': '2023-09-05T15:23:01.595214',
+ 'time_started': '2023-09-05T15:13:51.034780',
+ 'time_submitted': '2023-09-05T15:09:02.768518'}
+```
+
+### Access the predictions
+
+You can now directly access your predictions in your application.
+If you left the default option you can access them via the dynamically created
+collection with a query such as the following:
+
+```
+## Query to return results
+
+query = f"""
+FOR prediction IN {prediction_status.job_state_information['outputCollectionName']}
+    RETURN prediction
+"""
+results = db.aql.execute(query)
+
+for r in results:
+    for key in r:
+        print(key +": "+ str(r[key]))
+    print(" ")
+```
```

Comment:
(not a documentation request)

Just seeing this now for the first time. Perhaps we can make things easier for the user here? Offering this logic this as a one-liner would probably be ideal


maybe something like `arangoml.predictions.get_prediction_results(prediction_status)` ?